lib/modules/manager/composer/readme.md

```diff
@@ -1,3 +1,10 @@
+<!-- prettier-ignore -->
+!!! warning
+    Renovate does not support Composer version `2.5.0` or newer.
+    This is because of changes in the default behavior of Composer.
+    For now, you can do ... to workaround the problem.
```

Comment:
Need a suggestion from the maintainer on our recommended workaround. I suppose it's something like this?

- Pin Composer to last known working version (probably `2.4.4`)
- Don't merge Renovate upgrade for Composer `2.5.0` upgrade until Renovate works again